src/test/java/com/atlassian/bitbucket/jenkins/internal/status/BuildStatusPosterTest.java

```diff
@@ -15,6 +15,9 @@
 import org.mockito.junit.MockitoJUnitRunner.Silent;
```

Comment:
We should add a test for this class testing when multiple revision actions are present on a run, to ensure all are having their restriction sent through correctly.